### PR TITLE
Stories and personas 76

### PR DESCRIPTION
**Pull request by ThomasRusbjerg**

### CODE REVIEW

File: report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+% Factory owner
+The \textbf{factory owner} is responsible for the surrounding environment in which the system should be deployed. This person may present non-technical constraints that will effect the technical options of the system. Factory owners mostly have a financial stake in the system, and will likely not interact with the system with a specific purpose.
+
+% Machine responsible
+The \textbf{maintenance responsible} has the responsibility for the machine that data is being collected from. This person could be the operator of a given machine or a more general factory technician. This archetype will be using the system to monitor the status of devices in the factory and make sure that data is being collected. Since this type is operating at the factory they will likely take on the task of changing batteries on devices.
+
+% Installer
+The \textbf{installer} is responsible for installing the devices onto the physical machines at the factories. The installer does not necessarily have any technical know how and should not need to be able to configure devices. The installer simply mounts the devices.
```
Comment (Dbraae) [inline report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex]: The last sentence seems redundant.

```suggestion
The \textbf{installer} is responsible for installing the devices onto the physical machines at the factories.
The installer does not necessarily have any technical know-how and should not need to be able to configure devices.
```

File: report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+% Machine responsible
+The \textbf{maintenance responsible} has the responsibility for the machine that data is being collected from. This person could be the operator of a given machine or a more general factory technician. This archetype will be using the system to monitor the status of devices in the factory and make sure that data is being collected. Since this type is operating at the factory they will likely take on the task of changing batteries on devices.
+
+% Installer
+The \textbf{installer} is responsible for installing the devices onto the physical machines at the factories. The installer does not necessarily have any technical know how and should not need to be able to configure devices. The installer simply mounts the devices. 
+
+% Data analyst
+The \textbf{data analyst} use the collected data from the machines to verify whether the data can be used for predictive maintenance. They are only interested in extracting the data from the system in a appropriate way and have no interest in the devices themselves. They have preferences to the quality and quantity of data and therefore present technical constraints of the system.
```
Comment (Dbraae) [inline report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex]:
```suggestion
The \textbf{data analyst} uses the collected data from the machines to verify whether the data can be used for predictive maintenance.
They are only interested in extracting the data from the system in a appropriate way and have no interest in the devices themselves.
They have preferences to the quality and quantity of data and therefore present technical constraints for the system.
```

File: report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+% System designer
+The \textbf{system architect} is responsible for designing an instance of the system that fits a given use case. This includes choosing the number of devices and how they should be configured e.g. which sensors to use. The system architect makes devices ready for the installer, such that devices are easy to deploy.
```
Comment (Dbraae) [inline report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex]:
```suggestion
The \textbf{system architect} is responsible for designing an instance of the system that fits a given use case.
This includes choosing the number of devices and how they should be configured e.g. which sensors to use.
The system architect makes devices ready for the installer, such that devices are easy to deploy.
```

File: report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+An example of a user story can be seen in \autoref{fig:user-story-example}
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.6\textwidth]{PartProblemDomain/ChapterUserStoriesAndArchetypes/images/userstoryexample.png}
+	\caption{An example of a user stories selected for sprint 3.}
+	\label{fig:user-story-example}
+\end{figure}
+
+Duo to the agile nature of our development process were the user stories often redefined and more added though out the sprints. At the start of each sprint were a set of user stories chosen. These stories can be found in the Sprint Objective section of each sprint log, see \autoref{par:sprint-log}.
```
[the remark this answers is not in the document]
Comment (Dbraae) [reply report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex]: I don't think we should write here how we used the user stories. Just write that the ones we worked on can be found in the sprint logs.

[the remark this answers is not in the document]
Comment (ThomasRusbjerg) [reply report/PartProblemDomain/ChapterUserStoriesAndArchetypes/UserStoriesAndArchetypes.tex]: We added more. Which is normal btw

**Pull request merged**